infra/nginx/default.conf

```diff
@@ -55,6 +55,13 @@ server {
         proxy_set_header X-Real-IP $remote_addr;
     }
 
+    location /ui/v1 {
+        # TODO(marcel): create this service. Is the port ok?
```

Comment:
port is okay id like to map / to to the client, for all others routes besides /api and /auth.
But I'll change that myself when adding the client here 👍